docs/tutorial/index.md

```diff
@@ -289,24 +135,31 @@ admin account. Enter the following credentials and click the **Sign Up** button:
 
 ![New Landscape admin account creation](./assets/new-standalone-user.png)
 
-Finally, copy the Landscape server certificate into your Windows user profile directory. The Landscape client inside of any WSL
-instance will need that certificate to connect to the server.
+The Landscape client inside of any WSL instance will need the Landscape server certificate to connect to the server.
 
-```bash
+To achieve this copy the Landscape server certificate into your Windows user profile directory:
+
+```text
 root@mib:~# cp /etc/ssl/certs/landscape_server.pem /mnt/c/users/me/
-root@mib:~#
 ```
 
-Done -- your self hosted Landscape server is now up and running! At this point, if you configure the Landscape client on your Ubuntu WSL instances to know about this server, that will register them with the Landscape service included in your Ubuntu Pro subscription as well, and you'll be able to manage these instances at scale from your Landscape server.
+Done -- your self-hosted Landscape server is now up and running! 
+Now if you configure the Landscape client on your Ubuntu WSL instances to detect this server, they will also be registered with the Landscape service included in your Ubuntu Pro subscription.
 
-Keep the current terminal open so the server stays running while you continue on with this tutorial.
+The server will stay running until you close the terminal, so keep the terminal open for the rest of the tutorial.
+If you do close the terminal running `ubuntu2204.exe` in a new terminal window will start the Landscape server automatically.
 
-```{note}
-**If you accidentally close the terminal**:
+(tut::ensure-ubuntu-pro)=
+### Get an Ubuntu Pro token
 
-Open a new terminal window and run `ubuntu2204.exe`. Landscape server and related components will start automatically.
-```
+An Ubuntu Pro subscription gives you a token that can be added to the Ubuntu Pro client on WSL instances.
+
+Visit the [Ubuntu Pro](https://ubuntu.com/pro/subscribe) page to get a subscription.
+The `Myself` option for a personal subscription is free for up to 5 machines. 
 
+Your subscription token can then be retrieved from the [Ubuntu Pro Dashboard](https://ubuntu.com/pro/dashboard).
```

Comment:
If the user already has an active subscription (personal or business) accessing the "subscribe" page will prompt them to creata yet another subscription, which might cost money or be misleading if there is already an active subscription in place. So, I'm not so sure about this, but I think it's better to guide the user to the "dashboard" first, and in case they don't have an active subscription, then a redirection to the "subscribe" page will make sense. Feel free to interject on this.

Reply:
Good thinking. The text can be revised to include a statement to the effect that if you already have a subscription you can go straight to the dashboard, and if you don't you can get a subscription at the link. Of course, this will add a slight bit of friction for those who don't have a subscription but - importantly - it won't be costly or misleading.